node/typings/Search.ts

```diff
@@ -74,6 +75,7 @@ interface FacetsInput {
   fullText: string
   query: string
   searchState?: string
+  removeHiddenFacets: boolean
```

Comment:
```suggestion
  removeHiddenFacets?: boolean
```

Reply:
this parameter will always be received because in the schema I define that the default value is false